error-handling/src/main/java/com/palantir/remoting1/errors/RemoteException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting1.errors;
+
+/**
+ * An exception thrown to propagate a remote exception.
+ */
+public final class RemoteException extends RuntimeException {
```

Comment:
David, I would typically expect Jackson serialization to be used for this
class, not Java serialization.

On Sat, Aug 13, 2016, 17:52 David Schlosnagle <notifications@github.com>
wrote:

> In
> error-handling/src/main/java/com/palantir/remoting1/errors/RemoteException.java
> <https://github.com/palantir/http-remoting/pull/199#discussion_r74694178>:
>
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +package com.palantir.remoting1.errors;
> > +
> > +/**
> > + * An exception thrown to propagate a remote exception.
> > + */
> > +public final class RemoteException extends RuntimeException {
>
> serialVersionUid?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/199/files/bc2affd0f624fb975b783154957f8d3321ebc2c6#r74694178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwaKAxJJD42oycR4qX77zLDpS2Cl6ks5qfma0gaJpZM4Jjywx>
> .
>